htdocs/admin/mails.php

```diff
@@ -383,7 +386,7 @@ function initfields()
 	// PW
 	if (!empty($conf->use_javascript_ajax) || (isset($conf->global->MAIN_MAIL_SENDMODE) && in_array($conf->global->MAIN_MAIL_SENDMODE, array('smtps', 'swiftmailer'))))
 	{
-		$mainsmtppw = (!empty($conf->global->MAIN_MAIL_SMTPS_PW) ? $conf->global->MAIN_MAIL_SMTPS_PW : '');
+		$mainsmtppw = (!empty($conf->global->MAIN_MAIL_SMTPS_PW) ? md5($conf->global->MAIN_MAIL_SMTPS_PW) : '');
```

Comment:
Does this mean you will store the value in md5 in database ?

Reply:
@eldy no, i make a test to check if password is changed
it's to avoid that:
![image](https://user-images.githubusercontent.com/3624836/70426250-feb39f80-1a72-11ea-864b-28cbd9c6cc56.png)